### ISSUE TITLE
[FEATURE] Import from Fantasy Calendar

### ISSUE THREAD

**Issue by apoapostolov** [vigoren/foundryvtt-simple-calendar#58]
Perhaps the best, most powerful RP calendar tool there is, at https://app.fantasy-calendar.com/

Users who host their calendar there can export their complete calendar, which also includes holiday data when using popular RP setting template, so this is directly related to #52 issue.

How to get an export of a whole world with its events:
1. Register to their site.
2. Create a calendar with the desired template
3. When the calendar is created, at https://app.fantasy-calendar.com/calendars click on the ... menu and choose export. The export is presented as JSON format of the whole data, so picking the premade events in them is where you can get the holidays.

Overall supporting their import could allow to import existing calendars in Simple Calendar and allow to quickly get data for premade holiday calendars.

**Comment by vigoren** [comment]
Yeah that site is good tool. I can add this in with the entire import/export of Notes and settings.

**Comment by sturtus** [comment]
Seconded here: https://github.com/vigoren/foundryvtt-simple-calendar/issues/664
